api/http/api_deployments.go

```diff
@@ -973,6 +1043,18 @@ func ParseLookupQuery(vals url.Values) (model.Query, error) {
 
 	}
 
+	dType := vals.Get("type")
+	if dType == "" {
+		return query, nil
```

Comment:
Shouldn't we default to `query.Type = model.DeploymentTypeSoftware` here?
Not sure if it is used somewhere else, but to me it looks cleaner to set the type in this case.

Reply:
hmm, I thought that by default we'll return deployments of both types - software and configuration and we'll filter them only when the type is provided

Reply:
Sorry, you are right, I misread the diff: this is the parsing of the query, thus no default should be applied here.